frankenphp.go

```diff
@@ -428,11 +434,15 @@ func updateServerContext(request *http.Request, create bool, mrh C.uintptr_t) er
 		C.int(request.ProtoMajor*1000+request.ProtoMinor),
 	)
 
+	if rh == 0 {
+		rh = h
+	}
```

Comment:
Since the "smart-pointers" has gone away in this PR, we need to ensure we return the handle to delete at the end of this request.